### ISSUE TITLE
Things appear white through transparency.

### ISSUE THREAD

**Issue by SinsOfSeven** [SinsOfSeven/TexFx#13]
[Original Source](https://gamebanana.com/posts/12057187)
It seems that problems with transparent details will haunt us forever
![image](https://github.com/user-attachments/assets/5520a2d2-e434-4835-bd5b-e89cd931d86d)
![image](https://github.com/user-attachments/assets/cf8d53e7-b358-47c8-ac0f-2db3722d9f0e)

**Comment by SinsOfSeven** [comment]
This is caused by a new feature which I didn't thoroughly test and kind of forgot about, I will be pushing an update to remove it soon and reimplement it properly for the next major release. Basically, I didn't encounter this while testing, so I just left it in.

This issue is related to the mixing of the `o1` and `o4` outputs of the Pixel Shaders. The code can be found on these lines. I will temporarily remove this max function in favor of better handling in a future version. This change needs to be done to all 4 copies of the custom transparency shaders.

```hlsl
  r3.yz = float2(max(ren1.w,r3.y),max(ren4.x,r3.z));
  //...
  o1.w = r3.y;
  //...
  o4.x = r3.z;
```

From:
https://github.com/SinsOfSeven/TexFx/blob/85df8b678b352090e0a7d22b2954ba3543832806/hic_sunt_dracones/NatlanOutlineWithDiffuseColor1.hlsl#L221-L233


[NatlanOutlineWithDiffuseColor0.hlsl](https://github.com/SinsOfSeven/TexFx/blob/main/hic_sunt_dracones/NatlanOutlineWithDiffuseColor0.hlsl)
[NatlanOutlineWithDiffuseColor1.hlsl](https://github.com/SinsOfSeven/TexFx/blob/main/hic_sunt_dracones/NatlanOutlineWithDiffuseColor1.hlsl)
[OutlineWithDiffuseColor0.hlsl](https://github.com/SinsOfSeven/TexFx/blob/main/hic_sunt_dracones/OutlineWithDiffuseColor0.hlsl)
[OutlineWithDiffuseColor1.hlsl](https://github.com/SinsOfSeven/TexFx/blob/main/hic_sunt_dracones/OutlineWithDiffuseColor1.hlsl)

**Comment by SinsOfSeven** [comment]
oops forgot to say this is patched in the sub-release of 1.05108, but that's just a roll-back.
